proxy/ParentSelection.cc

```diff
@@ -354,7 +354,7 @@ ParentRecord::PreProcessParents(const char *val, const int line_num, char *buf,
 
   strncpy(_val, val, strlen(val));
 
-  token = strtok_r(_val, ";", &savePtr);
+  token = strtok_r(_val, ";, ", &savePtr);
```

Comment:
If you're going to go to this much trouble, you should really have a `constexpr char PARENT_DELIMITERS[] = ";, ";` to make it very clear these should all be the same.